### PR TITLE
SIMD is no longer in JavaScript (moved to WebAssembly).

### PR DESCRIPTION
**Pull request by brianblakely**
Per TC39 documentation: https://github.com/tc39/ecmascript_simd

### CODE REVIEW

**Review by ljharb — CHANGES_REQUESTED** [review]
The proposal is inactive, but the feature still exists. Instead of deleting it, please move it to nonstandard.

File: data-non-standard.js [hunk 1]
```diff
@@ -208,7 +1398,7 @@ exports.tests = [
   exec: function () {
     return (function () {}).arity === 0 &&
       (function (x) { return x; }).arity === 1 &&
-      (function (x, y) { return y; }).arity === 2;
+      (function (y) { return y; }).arity === 2;
```
Comment (ljharb) [inline data-non-standard.js]: this is incorrect; the arity should be 2 only when there's 2 arguments.

**Review by ljharb — CHANGES_REQUESTED** [review]

**Review by chicoxyzzy — CHANGES_REQUESTED** [review]
yes, please move it to non-standard page

**Review by chicoxyzzy — COMMENTED** [review]
Oops. Now I see you did it already. Please make changes according to @ljharb's comments

**Review by chicoxyzzy — APPROVED** [review]
Oh. I should work little bit less to prevent burnout :)
LGTM

File: data-non-standard.js [hunk 2]
```diff
@@ -208,7 +1398,7 @@ exports.tests = [
   exec: function () {
     return (function () {}).arity === 0 &&
       (function (x) { return x; }).arity === 1 &&
-      (function (x, y) { return y; }).arity === 2;
```
Comment (ljharb) [inline data-non-standard.js]: this function is fine continuing to return `y`; i'm not sure why this line needs a change at all. can you revert it?

**Review by ljharb — CHANGES_REQUESTED** [review]
One more change, and it'd be great if you could rebase down to one commit :-)

**Review by ljharb — APPROVED** [review]

**Pull request merged**